tests/test_repos.py

```
@@ -0,0 +1,84 @@
+            "star_count": 10,
+            "commit_count": 50,
+            "fork_count": 5,
+            "branch_default": "main",
+            "created_at": "2024-01-01T00:00:00Z",
+            "updated_at": "2024-01-01T00:00:00Z",
+        }
+    ]
+
+
+@pytest.fixture
+def sample_repos():
```

Ruff (0.11.9)

Redefinition of unused `sample_repos` from line 8

(F811)